src/index.js

```diff
@@ -26,7 +26,14 @@ export default (config) => {
       },
       gtagOptions: {},
     },
-    modules: ['tracking', 'youtube', 'facebook', 'google'],
+    modules: [
+      'tracking',
+      'youtube',
+      'facebook',
+      'google',
+      'twitter',
+      'instagram',
+    ],
```

Comment:
@danalvrz Let's leave twitter and instagram out of the default list. The others are included because they are supported by core volto blocks (video and maps block). We need to be more careful with backwards compatibility for this repo than for volto-social-blocks, since this one is already used in production sites.